client/src/pages/Login.js

```diff
@@ -0,0 +1,121 @@
+import React, { Fragment } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import { Toolbar } from '@material-ui/core';
+import { NavLink } from 'react-router-dom';
+import useForm from '../components/useForm';
+
+import Copyright from "../components/Copyright";
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+  link: {
+    color: "#DF1B1B",
+    fontSize: "0.75rem",
```

Comment:
Since the styling for this component and the signup component are the same, let's extract the style and import it in both components

Reply:
Done!